spyder_line_profiler/tests/test_lineprofiler.py

```diff
@@ -17,7 +17,11 @@
 MAIN_APP = qapplication() 
 
 # Local imports
-from spyder_line_profiler.widgets.lineprofiler import LineProfilerWidget
+import spyder
+import spyder_line_profiler
+from spyder_line_profiler.spyder.widgets import SpyderLineProfilerWidget
+from spyder_line_profiler.spyder.widgets import PythonModulesComboBox
```

Comment:
```suggestion
from spyder_line_profiler.spyder.widgets import SpyderLineProfilerWidget
```

These three imports don't seem necessary.